src/index.css

```diff
@@ -1,13 +1,57 @@
+@import url('https://fonts.googleapis.com/css?family=Abril+Fatface|Poppins');
+
 body {
+  box-sizing: border-box;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  padding: 0;
+  overflow-x: hidden;
+  width: 100vw;
+  font-family: "Poppins", sans-serif;
+  height: 100vh;
+  color: #fff;
+  background-image: url('./assets/bg3.jpg');
+  background-size: cover;
+}
+
+.container {
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.detail-container {
+  display: flex;
+  flex-direction: column;
+  width: 90vw;
+  padding: 0 50px 50px 50px;
+  flex-grow: 1;
+  align-self: center;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+.back {
+  padding: 50px 50px 0 50px;
+}
+
+@media only screen and (max-width: 768px) {
+  .container {
+    width: 100vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .detail-container {
+    display: flex;
+    flex-direction: column;
+    width: 90vw;
+    padding: 0 50px 50px 50px;
+    flex-grow: 1;
+  }
+
```

Comment:
- [ ] Like the previous reviewer mentioned kindly make sure you're following the layout of the design 🙏 
- The `Country flag` and `Country Name` should be on the same line 🙏 
- The `properties` and `values` of the country should be spaced out more as seen in the original design 🙏 

Original Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/190852397-7b2a8204-a8a8-4de0-9f65-44d57744b7fe.png)  |  ![image](https://user-images.githubusercontent.com/74814780/190852430-3a4813a6-eb1c-4904-b3e1-cb9a5feb8d2a.png)